news/2022-07-08-the-roundtable.md

```diff
@@ -0,0 +1,72 @@
+---
+
+16 osu! warriors battle in the inaugural edition of **The Roundtable**, osu!'s first major in-person invitational tournament!
+
+![](/wiki/shared/news/2022-07-08-the-roundtable/roundtable_banner.png)
+
+While LAN events are nothing new to the osu! scene, [**The Roundtable**](/wiki/Tournaments/The_Roundtable) takes a different approach.
+
+This **3 day** event from July 8-10 features **3 different tournaments**, all streamed from [BTMC](https://osu.ppy.sh/users/3171691)'s [Twitch channel](https://twitch.tv/BTMC) with comfy couch commentary and personal interactions with everyone involved!
+
+## Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| [16 player battle royale](/wiki/Tournaments/The_Roundtable/16_player_battle_royale) | 2022-07-06 (21:00 UTC +0) |
+| [2v2 team tandem tournament](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament) | 2022-07-07 (18:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) | 2022-07-07 (22:00 UTC +0) |
+| [1v1 double elimination tournament](/wiki/Tournaments/The_Roundtable/1v1_double_elimination) (top 8) | 2022-07-08 (17:30 UTC +0) |
+
+### 16 player battle royale
+
+Kicking off **The Roundtable** is a [**16 player battle royale**](/wiki/Tournaments/The_Roundtable/16_player_battle_royale)!
+
+The LAN setup has 8 computers, so the players will be randomized into two groups of 8. Each group will play **7** maps in a multiplayer lobby, and the lowest scorer of each map will be eliminated.
+
+Once both groups have been narrowed down to a single player, the finalists will face head to head in a **best of 13 competition!**
+
+### 2v2 team tandem tournament
+
+The event's second day launches with an unorthodox [**2v2 team tandem tournament**](/wiki/Tournaments/The_Roundtable/2v2_team_tandem_tournament).
+
+Honestly speaking, this is more about having fun than competition. "Tandem" in this context means there will be **two players clicking circles on one setup** — one aiming and the other tapping!
+
+Players will form four groups of four and face off in a single elimination tournament -- until the last group remains as the victor.
+
+### 1v1 double elminiation tournament
+
+The main event of **The Roundtable** is a high stakes [**1v1 double elimination tournament!**](/wiki/Tournaments/The_Roundtable/1v1_double_elimination).
+
+This tournament will last two days: The first day of matches will determine the Top 8 players, while the second day is reserved for illustrating the talents of the best players at **The Roundtable.**
+
+This classic 1v1 has the largest set of prizes, so expect to see players perform - in person - at the highest caliber.
+
+## Players
+
+So who's coming to this thing anyways?
+
+**15** of North America's strongest competitors have been invited, including former and current [OWC](/wiki/Tournaments/OWC) champions.
+
+Take a peak at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player:
```

Comment:
Take a peek at everything at the full list below, including a special teaser reveal for osu!'s rank 1 player: